pkg/install/resources.go

```diff
@@ -226,11 +225,6 @@ func AllCRDs(perferredAPIVersion string) *unstructured.UnstructuredList {
 	resources.SetGroupVersionKind(schema.GroupVersionKind{Group: "", Version: "v1", Kind: "List"})
 
 	switch perferredAPIVersion {
-	case "v1beta1":
```

Comment:
Per my understanding after moving to kubebuilder v3, all CRDs of velero will be v1, so the `preferredAPIVersion` is unnecessary?

Reply:
In release 1.8, there is only one possible version, v1, but preferredAPIVersion is passed from velero client command line parameter "--crds-version". 
I think it may be useful for future potential multiple versions need. 
In my opinion, keeping the logic there and adding validation to only support v1 may be better.

Reply:
Please check with @zubron 
By checking the commit:
https://github.com/vmware-tanzu/velero/commit/fca2b5c417ed87facb08038273d6af5ccd46b524

It seems this was added as a temp workaround.

If the conclusion is to remove this param I'm fine to handle that in another PR/issue so that this big PR is not blocked.

Reply:
Yes, this flag was only in place to allow users to explicitly set which version to use if needed. Given that we always use v1 now, you can remove this flag.

There is also some code in the uninstall path to remove v1beta1 CRDS (https://github.com/vmware-tanzu/velero/blob/main/pkg/cmd/cli/uninstall/uninstall.go#L139-L150) but I think we may want to keep that for now to handle any v1beta1 CRDs that a user may have installed. We can probably remove that in 1.9.

Reply:
Thanks. I think the agreement is to remove the CRD version selection parameter in install command, and keep the v1beta1 check in uninstall command until 1.9 release.